views/post-view/src/components/Post/index.tsx

```
@@ -157,22 +130,16 @@ export default function Post({
 
       {hasBody && (
         <j-box pt="500">
-          <div
-            className={styles.postBody}
-            dangerouslySetInnerHTML={{ __html: post.body }}
-          />
+          <div className={styles.postBody} dangerouslySetInnerHTML={{ __html: post.body }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)